android/app/src/main/java/org/fossasia/openevent/utils/SortOrder.java

```diff
@@ -48,7 +48,7 @@ public static String sortOrderSchedule(Context context) {
                 return DbContract.Sessions.START_TIME;
             case SORT_TYPE_SECOND:
                 //By TRACKS
-                return DbContract.Sessions.TITLE;
+                return DbContract.Sessions.TRACK;
```

Comment:
Why do you want to do this?
I think that the current sort types are good enough.